packages/components/src/components/primitives/Toggle/Toggle.types.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseProps } from 'components/Components.types';
+
+export type Props = BaseProps & {
+  enabled?: boolean;
```

Comment:
on instead?, having enabled and disabled as properties ...

Reply:
Renamed